.github/CODEOWNERS

```diff
@@ -1,39 +1,39 @@
 
 /tools/ @zikalino
 /scripts/ @zikalino
 /src/azure-cli-testsdk/ @zikalino
 /src/azure-cli-core/ @yugangw-msft
-/src/command_modules/azure-cli-acr/ @djyou
-/src/command_modules/azure-cli-acs/ @rjtsdl
-/src/command_modules/azure-cli-advisor/ @Prasanna-Padmanabhan
-/src/command_modules/azure-cli-appservice/ @yugangw-msft
-/src/command_modules/azure-cli-backup/ @dragonfly91
-/src/command_modules/azure-cli-batch/ @bgklein
-/src/command_modules/azure-cli-batchai/ @AlexanderYukhanov
-/src/command_modules/azure-cli-cosmosdb/ @dmakwana
-/src/command_modules/azure-cli-cloud/ @yugangw-msft
-/src/command_modules/azure-cli-container/ @samkreter
-/src/command_modules/azure-cli-consumption/ @sandeepnl
-/src/command_modules/azure-cli-dls/ @lewu-msft
-/src/command_modules/azure-cli-eventgrid/ @kalyanaj
-/src/command_modules/azure-cli-extension/ @yugangw-msft
-/src/command_modules/azure-cli-keyvault/ @yugangw-msft
-/src/command_modules/azure-cli-monitor/ @yugangw-msft
-/src/command_modules/azure-cli-natgateway/ @khannarheams
-/src/command_modules/azure-cli-network/ @MyronFanQiu
-/src/command_modules/azure-cli-policyinsights/ @cheggert
-/src/command_modules/azure-cli-profile/ @marstr
-/src/command_modules/azure-cli-resource/ @adewaleo
-/src/command_modules/azure-cli-role/ @marstr
-/src/command_modules/azure-cli-storage/ @limingu
-/src/command_modules/azure-cli-servicefabric/ @QingChenmsft
-/src/command_modules/azure-cli-sql/ @jaredmoo
-/src/command_modules/azure-cli-vm/ @adewaleo
-/src/command_modules/azure-cli-eventhubs/ @v-ajnava
-/src/command_modules/azure-cli-servicebus/ @v-ajnava
+/src/azure-cli/azure/cli/command_modules/acr/ @djyou
+/src/azure-cli/azure/cli/command_modules/acs/ @rjtsdl
+/src/azure-cli/azure/cli/command_modules/advisor/ @Prasanna-Padmanabhan
+/src/azure-cli/azure/cli/command_modules/appservice/ @yugangw-msft
+/src/azure-cli/azure/cli/command_modules/backup/ @dragonfly91
+/src/azure-cli/azure/cli/command_modules/batch/ @bgklein
+/src/azure-cli/azure/cli/command_modules/batchai/ @AlexanderYukhanov
+/src/azure-cli/azure/cli/command_modules/cosmosdb/ @dmakwana
+/src/azure-cli/azure/cli/command_modules/cloud/ @yugangw-msft
+/src/azure-cli/azure/cli/command_modules/container/ @samkreter
+/src/azure-cli/azure/cli/command_modules/consumption/ @sandeepnl
+/src/azure-cli/azure/cli/command_modules/dls/ @lewu-msft
+/src/azure-cli/azure/cli/command_modules/eventgrid/ @kalyanaj
+/src/azure-cli/azure/cli/command_modules/extension/ @zikalino
+/src/azure-cli/azure/cli/command_modules/keyvault/ @jiasli
+/src/azure-cli/azure/cli/command_modules/monitor/ @MyronFanQiu
+/src/azure-cli/azure/cli/command_modules/natgateway/ @khannarheams
+/src/azure-cli/azure/cli/command_modules/network/ @MyronFanQiu
+/src/azure-cli/azure/cli/command_modules/policyinsights/ @cheggert
+/src/azure-cli/azure/cli/command_modules/profile/ @jiasli
+/src/azure-cli/azure/cli/command_modules/resource/ @Juliehzl
+/src/azure-cli/azure/cli/command_modules/cli-role/ @jiasli
```

Comment:
I cannot find this path. Looks like it should be `/src/azure-cli/azure/cli/command_modules/role/`

Reply:
Good catch.